tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "commonjs",
+    "strict": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true
```

Comment:
This requires adding some new settings to TS for using this lib, and I think it should be avoided.

Reply:
I can avoid this, by importing a minimal openapiv3 spec from a .ts file instead of a .json file.